### DISCUSSION TITLE
PiKVM updated, no longer able to use JackTrip?

### DISCUSSION THREAD

**Discussion by MarkHowell1961** [jacktrip/jacktrip#1234] [General]
I woke up this morning to not being able to get sound through my PiKVM. I ended up having to uninstall JackTrip to get it to work again.  Using (arch arm linux), got this error: "Installing rtaudio(6.0.0.1-1) breaks dependency 'librtaudio.so=6-32' required by jacktrip. 

If I try to install jacktrip, I get the following:
![image](https://github.com/jacktrip/jacktrip/assets/106042755/4b7dcf24-8a06-4ab5-a930-6419d791b65a)

Looking at the ARM, I can't seem to find a solution. Trying to search rtaudio within: https://archlinuxarm.org/packages

![image](https://github.com/jacktrip/jacktrip/assets/106042755/809a6a38-e71d-4704-886f-d0805c9552df)

Would anyone know how to go about this?

**Comment by mikedickey** [comment]
Possibly this? https://github.com/thestk/rtaudio/issues/405

Does it work ok when you use rtaudio 6.0.1?
